### ISSUE TITLE
Joystream Community Handbook: Improved & Unified Community Documentation

### ISSUE THREAD

**Issue by bedeho** [Joystream/joystream#2894]
# Background

Where to begin 😶‍.

Currently, the set of information that active participants, or prospective active participants, in the joystream testnets have to consume is

1. distributed across a range of venues:
   - https://github.com/Joystream/helpdesk <== extremely dense.
   - https://github.com/Joystream/community-repo
   - https://github.com/Joystream/founding-members/
   - https://github.com/Joystream/joystream-content-system <== what is this?
   - https://blog.joystream.org/sumer-kpis/
   - https://github.com/Joystream/design
   - https://joystream.gitbook.io/joystream-handbook/ <== not so much today, but in the future?
   - anything else?
2. presented in experiences which are tailored for presenting information for non-developers: Github 👀 
3. not internationalized
4. not organized in a way that optimized for the reader <= mostly a guess, but pretty sure
5. is near impossible for outsiders to contribute to, because Git is very difficult for non-technical people.
6. no social collaboration or questions directly in the content.
7. rich content is difficult, or impossible.

# Proposal

- Place all information needed by users into a singular space, which I am calling `Joystream Community Handbook`: Gitbook or Notion are the major alternatives. I would say Gitbook is the best place to try now, as it already supports markdown very deeply, no manual work needed, and also works off of Git, which is sort of nice for the possible future use of this data, while Notion is a fully closed model.
- Try to reconcile the actual content into one coherent structure which can sensibly be understood. I don't think this venue should be understood as a work to be read from start to finish, rather its a reference resource, so people will have some context when they try to look at something, but the overall structure must make sense.
- Importantly, we may not need to put all information I found above into this venue, in particular the existing joystream handbook does not make sense, because it is extremely technical, hence its just going to be noise for 99.9% of all visitors. We can easily combine later if that makes sense later, but not now. In the same spirit, all other references in the list should also be evaluated from this lens. Perhaps some of them make sense splitting up.
- The reorganization should really be informed by trying to keep some narrower set of target readers in mind for a given piece of content. This means we should absolutely consider throwing stuff away, rewriting it, or adding totally new things. One thing that is very clearly missing yo me is step-by-step tutorials, with **screen shots** from Pioneer 2 & CLI, of how to do certain things in the system, such as applying for the council, or applying for a job, or making a proposal, or any number of things.

# Process

- I think the best way to make initial progress on this is to review all of the content, think about the full state of the Joystream system, products and Jsgenesis initiatives, and then come up with some sort of table of contents for the entire thing, where one explains the goal of every piece of content, before we start actually writing anything.
- We should try to identify opportunities for what the community can contribute here, in particular when it comes to content or initiatives we do not have time to do, for example support in other languages. In fact, the whole impetus for this project came from a user who had started work to do this on their own, because they were so confused https://mmaguadalajara.gitbook.io/joystream/.
